src/pages/ps_reference/changelog/index.md

```diff
@@ -131,6 +109,42 @@ The CountItems DOM API is now available in Photoshop.
   - `move()`, `remove()`
   - 
 
+  
+### UXP 
+From the [UXP team](../../uxp/) we have integrated the following update.
+
+
+#### **v6.4 Integration**
+
+#### New
+- Webviews for Panels
+- Support Path Module APIs
+- Script can run fsapi with full access in localFileSystem
+
+#### Bugs Fixes
+- PluginManifest parsing / Load Plugin errors not logged in UDT
+- Error shown in UDT APP logs on Plugin UnLoad
+- UXP script : file session tokens are not getting created
+- Error message and stack are empty in case of reject("error string") executed in script
+- [Win only]-Volume is being muted automatically for 3rd video in the tutorial while opening the video in Theatre mode
+- After closing the theatre mode, video's are Overlapping while scrolling up and down in DP
+- Provide an API to programmatically close dialog in pop-out/theatre mode.
+- Plugin Data folder shouldn't be created for scripts
+- Crash on exit
+- script.setResult("kSuccess") not working
+- Resource Leaks in Window Dialog on UWP
+- Privilege plugin fail to add in XD in certain cases
+- Fixed mouse/keyboard events within Webview in XD v55 and Win 10.
+
+
+#### **v6.5 Integration** 
```

Comment:
why is 6.5 integration changelog below 6.4? Are these docs generated?

Reply:
UXP v6.5 details should be in PS 24.2 beta section. I have corrected this.